src/compiler/compile/nodes/Element.ts

```diff
@@ -444,6 +449,13 @@ export default class Element extends Node {
 				});
 			}
 
+			if (react_attributes.has(attribute.name)) {
+				component.warn(attribute, {
+					code: 'use-the-platform',
+					message: `'${attribute.name}' is not a valid html attribute. Did you mean '${react_attributes.get(attribute.name)}'? If you are a recovering React developer, remember, Svelte actually uses the web platform.`
```

Comment:
As funny as it is, I'm not sure about this react "insult". I'm in favor of dropping that line, or at least drop the react reference.

Reply:
i just copied the warning message from the issue, i know it's a joke, i was being lazy and hoping someone would comment and propose a better warning message

Reply:
It'd probably be fine to just end it after the question mark

Reply:
how abut the `warning` code, don't think should use the `"use-the-platform"`

Reply:
`invalid-html-attribute`?